projects/core/directives/dropdown/dropdown-position.directive.ts

```diff
@@ -14,7 +14,7 @@ import {TuiDropdownDirective} from './dropdown.directive';
 import {TUI_DROPDOWN_OPTIONS, type TuiDropdownAlign} from './dropdown-options.directive';
 
 @Directive({
-    standalone: true,
+    selector: '[tuiDropdown]:not(ng-container):not(ng-template)',
```

Comment:
Same as for `TuiDropdownDirective`

https://github.com/taiga-family/taiga-ui/blob/c9a79d6da5505512a486d66ae9ba62a1b14de230/projects/core/directives/dropdown/dropdown.directive.ts#L39